site/docs/sql/imap-connector.md

```diff
@@ -128,10 +128,25 @@ the class using reflection and use its properties as column names. It
 recognizes public fields and JavaBeans-style getters. If some property
 has a non-primitive type, it will be mapped under the `OBJECT` type.
 
-## External Name
+## External Mapping Name
```

Comment:
I think we can omit this section. It's clear enough from the common documentation... Also for Kafka